docs/report_identification.md

```diff
@@ -0,0 +1,28 @@
+# Analyzer report identification
+
+Unique report idetifiers are required to compare analysis results.
+If the static analyzers (Clang Static Analyzer, Clang Tidy) do not generate
```

Comment:
Please highlight tool names with backtick: `Clang Static Analyzer`, `Clang Tidy`

Reply:
If the ~~static~~ analyzers